APIManager.js

```diff
@@ -0,0 +1,67 @@
+//This is the class that will manage all your APIs
+class APIManager {
+    constructor() {
+      this.data = {};
+    }
+    loadData() {
+      //get user
+      $.ajax({
+        method: "GET",
+        url: "https://randomuser.me/api/?results=7&inc=name,location,picture",
+        dataType: "json",
+        success: (data) => {
+          this.data["user"] = data.results[0];
+          // render.renderUser({ user: data.results[0] });
+          data.results.splice(0, 1);
+          const friends = data.results.map((d) => {
+            return { name: d.name };
+          });
+          this.data["friends"] = friends;
+        },
+        error: function (xhr, text, error) {
+          alert(
+            "Uh oh, something has gone wrong. Please tweet us @randomapi about the issue. Thank you."
+          );
+        },
+      });
+      //get Quote
+      $.ajax({
+        method: "GET",
+        url: "https://api.kanye.rest",
+        success: (data) => {
+          this.data["quote"] = data.quote;
+        },
+        error: function (xhr, text, error) {
+          alert("Uh oh, something has gone wrong.");
+        },
+      });
+      //get Pokemon
+      const i = Math.floor(Math.random() * 100)
```

Comment:
maybe call this `id` since that is the query that the pokemon api asks for. Will make reading your code easier if you use variables that describes the value well.